RELEASING.md

```diff
@@ -129,7 +129,7 @@ Please see [opensearch-project/.github#81](https://github.com/opensearch-project
 
 ## Backporting
 
-This project follows [semantic versioning](https://semver.org/spec/v2.0.0.html). Backwards-incompatible changes always result in a new major version and will __never__ be backported. Small improvements and features will be backported to a new minor version (e.g. `2.9.0`). Security fixes will be backported to a new patch version (e.g. `2.9.1`).
+This project follows [semantic versioning](https://semver.org/spec/v2.0.0.html). Backwards-incompatible changes always result in a new major version and will __never__ be backported. Small improvements and features will be backported to a new minor version (e.g. `2.9.0`). Security fixes and backward compatible bug fixes will be backported to a new patch version (e.g. `2.9.1`).
```

Comment:
Should we remove the word "small" from "small improvements"?

I would remove "backward compatible" from "backward compatible bug fixes" because in a 2.11 we would fix a bug introduced in 2.10, that changed previously working behavior from 2.9.

Reply:
I was thinking of aligning this section with [Semvar](https://semver.org/) to avoid further confusion. According to SemVar "PATCH version should incorporate backward compatible bug fixes" only.

Reply:
I see, makes sense.